mozilla/fx-private-relay
# MPP-3838: restore safer CSP

Use a new EagerNonceCSPMiddleware to add nonce to the CSP and update the React app to include it in dynamic scripts.

This PR fixes MPP-3838.

How to test:
1. Open [the Relay GA4 Realtime view](https://analytics.google.com/analytics/web/#/p314403930/realtime/overview?params=_u..comparisons%3D%5B%7B%22name%22:%22Hostname%20exactly%20matches%20dev.fxprivaterelay.nonprod.cloudops.mozgcp.net%22,%22isEnabled%22:true,%22filters%22:%5B%7B%22fieldName%22:%22hostname%22,%22expressionList%22:%5B%22dev.fxprivaterelay.nonprod.cloudops.mozgcp.net%22%5D,%22isCaseSensitive%22:true%7D%5D%7D%5D%26_u..nav%3Dmaui&collectionId=user)
2. Check out this branch locally
3. In `frontend/`, `npm run watch` to build the new front-end files
4. `python manage.py runserver`
5. Visit http://127.0.0.1:8000/ and perform some page views & events
   * [ ] Check that your new events show up in the Relay GA4 Realtime view
